# Importing required Libraries

In [3]:
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import pandas as pd

In [4]:
haber = pd.read_csv('haberman.csv', names=['Age', 'Year', 'Nodes', 'Status'])
haber.head()

Age  Year  Nodes  Status
0   30    64      1       1
1   30    62      3       1
2   30    65      0       1
3   31    59      2       1
4   31    65      4       1

# Ans 1

In [5]:
print(haber.shape)

(306, 4)


In [9]:
haber['Status'].value_counts()

1    225
2     81
Name: Status, dtype: int64

### Observation
So,1) There are 306 input points

2) There are 3 features i.e. Age, Nodes and Year

3)There is only one class i.e. Patient's survival Status

4) data-points per class for survival value = '1' - 225
                             survival value = '2' - 81

# Ans 2